src/stylesheets/core/mixins/utilities/_padding.scss

```diff
@@ -8,18 +8,18 @@
     $important: ' !important';
   }
   @if $side == all {
-    padding: units($value...)#{$important};
+    padding: get-uswds-value(padding, $value...)#{$important};
```

Comment:
This seems fine, but I'm not sure I understand why this change was made. Can you give me a little context?

Reply:
Mostly to make it consistent with the other utility mixins that are built with the same function. It handles errors a little differently — that is, `get-uswds-value` accepts non-USWDS values (like `17px`) with a warning and `units` throws an error. `get-uswds-value()` also "knows" the standard values for padding and will reject something like `neg-2` or `-2` since those negative values are invalid for padding.